<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>端口号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_3"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>校验位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_4"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>截取字符</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_5"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox_6"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="comboBox_7"/>
   </item>
   <item row="0" column="0" rowspan="7">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>关闭串口</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
